changelog/fix-4071-text-placeholders-not-working-on-in-person-payment-receipt-email

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix placeholder injection problem in New Receipt email
```

Comment:
I'm not the best person to judge what is or isn't good changelog entry but `injection problem` seems quite generic.

Reply:
Done